### PR TITLE
Yank aoc subscribe

### LINKED ISSUE #970: yank role commands
## Description

Remove the subscribe commands from lance, because as of https://github.com/python-discord/bot/pull/1868, this is now supported on Bot. (Beautifully executed, too)

EDIT: Rather than removing the commands, they will be replaced with a message that tells the user to use the command on bot.

## Reasoning

No sense in having two commands that do the same thing, more maintenance work!

## Proposed Implementation

**TODO:**
- [x] remove lovefest role command
- [ ] remove advent of code command (deferred until January, when the command can no longer be assigned or removed)

## Would you like to implement this yourself?

- [x] I'd like to implement this feature myself
- [ ] Anyone can implement this feature

### PR DESCRIPTION
**Pull request by onerandomusername**
### This is now safe to merge

Depends on #971, where the initial implementation is.
## Relevant Issues

Partially closes #970

## Description

Yanked the aoc subscribe commands. See #971 for how this works.
## Did you:

- [x] Join the [**Python Discord Community**](https://discord.gg/python)?
- [x] Read all the comments in this template?
- [x] Ensure there is an issue open, or link relevant discord discussions?
- [x] Read and agree to the [contributing guidelines](https://pythondiscord.com/pages/contributing/contributing-guidelines/)?

### CODE REVIEW

File: bot/constants.py [hunk 1]
```diff
@@ -23,6 +23,7 @@
     "Reddit",
     "RedisConfig",
     "RedirectOutput",
+    "PYTHON_PREFIX"
```
Comment (burakhanaksoy) [inline bot/constants.py:26]: I just started on this project, AFAIK, `__all__` is used for wrapping classes under constants.py.

If true, we can remove `PYTHON_PREFIX` from `__all__`

**Comment by onerandomusername** [comment]
Rebased onto main to remove the implementation commit, as that was added with #971

File: bot/exts/events/advent_of_code/_cog.py [hunk 2]
```diff
@@ -63,44 +69,28 @@ async def adventofcode_group(self, ctx: commands.Context) -> None:
...
+        This command has been moved to bot, and will be removed in the future.
+        """
+        raise MovedCommandError(MOVED_COMMAND)
 
     @in_month(Month.DECEMBER)
     @commands.guild_only()
-    @adventofcode_group.command(name="unsubscribe", aliases=("unsub",), brief="Notifications for new days")
+    @adventofcode_group.command(name="unsubscribe", aliases=("unsub",), help=MOVED_HELP)
     @whitelist_override(channels=AOC_WHITELIST)
     async def aoc_unsubscribe(self, ctx: commands.Context) -> None:
```
Comment (ChrisLovering) [inline bot/exts/events/advent_of_code/_cog.py:87]: Please just remove this command entirely and add it to the aliases of the other command. No need having two commands that do the same thing.

**Review by ChrisLovering — CHANGES_REQUESTED** [review]

File: bot/exts/events/advent_of_code/_cog.py (hunk 2 above)
Comment (onerandomusername) [reply bot/exts/events/advent_of_code/_cog.py]: https://github.com/python-discord/sir-lancebot/pull/972/commits/a309e6fbd497fe493ce6fca484c1d1ec20d2fb26

File: bot/exts/events/advent_of_code/_cog.py [hunk 3]
```diff
@@ -26,6 +29,9 @@
 # They aren't spammy and foster discussion
 AOC_WHITELIST = AOC_WHITELIST_RESTRICTED + (Channels.advent_of_code,)
 
+MOVED_COMMAND = f"{PYTHON_PREFIX}subscribe"
+MOVED_HELP = f"NOTE: This command has been moved to {MOVED_COMMAND}"
+
```
Comment (ChrisLovering) [inline bot/exts/events/advent_of_code/_cog.py:34]: No need for these to be variables, as they're only used in one place now

**Review by ChrisLovering — CHANGES_REQUESTED** [review]

File: bot/exts/events/advent_of_code/_cog.py (hunk 3 above)
Comment (onerandomusername) [reply bot/exts/events/advent_of_code/_cog.py]: https://github.com/python-discord/sir-lancebot/pull/972/commits/59b2993c51d93b2af7d48f386696eceb3379d1fa

**Review by ChrisLovering — APPROVED** [review]

**Comment by onerandomusername** [comment]
There were merge conflicts, so merged with main.

**Review by Xithrius — APPROVED** [review]
yuh

**Pull request merged**